src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -58,10 +58,10 @@ static ColliderCylinderInit sCylinderInit = {
 static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
 
 typedef enum {
-    /* 0 */ ENMA1_ANIM_0,
-    /* 1 */ ENMA1_ANIM_1,
-    /* 2 */ ENMA1_ANIM_2,
-    /* 3 */ ENMA1_ANIM_3
+    /* 0 */ ENMA1_ANIM_NOMORPH,
+    /* 1 */ ENMA1_ANIM_IDLE,
+    /* 2 */ ENMA1_ANIM_SING_NOMORPH,
+    /* 3 */ ENMA1_ANIM_SING
```

Comment:
I think `ENMA1` -> `MALON_` is fine, considering it's local to the file it shouldn't be confused with the other actors